### DISCUSSION TITLE
how often are Wikipedia POI location data updated?

### DISCUSSION THREAD

**Discussion by vapier** [osmandapp/OsmAnd#12744] [Q&A] [answered]
I'm looking at this POI:
https://en.wikipedia.org/wiki/House_at_72R_Dane_Street

the Wikipedia article has the correct coordinates:
42°22′45.8″ N 71°6′13.8″ W

but osmand has:
42°22′34.0″ N, 71°06′16.0″ W

that difference in a few seconds moves the location wildly -- more than 300m which is a few city blocks.

osmand doesn't seem to have a way to fix it in the app, nor a way to report it, and I don't know where the app is getting its data. if it scrapes Wikipedia, it seems the data is quite stale. the article had its coordinates updated in Jan 2021.

**Comment by vshcherb** [comment, marked as answer]
It's updated regularly every quarter / half year. OsmAnd uses coordinates from Wikidata to avoid dealing issues with different languages - https://www.wikidata.org/wiki/Q5914459. So the coordinates are not correct on Wikipedia in that sense

**Reply by vapier** [reply]
thanks, i hadn't seen Wikidata before. i fixed the coordinates there. not sure how to make Wikipedia use Wikidata's coordinates values to avoid duplication, but oh well.
